shell/platform/fuchsia/dart_runner/embedder/builtin.dart

```diff
@@ -25,7 +24,7 @@ class _Logger {
 }
 
 @pragma('vm:entry-point')
-String _rawScript;
+late String _rawScript;
```

Comment:
What is the distinction between the `late` vm entrypoint fields and the nullable ones, is this guaranteed to be set, while the others might still be null?

Reply:
Yeah... all of the other entry points could be null since they are set by the engine and do have real reasons for being null. The view ref, for example, is only available for things that need a UI. The other entry points will cause the program to not function well when they are null be we currently rely on them being null for host side tests.

I couldn't find a valid reason why this value would be null. If there is one I can update this to be `?` instead of `late`.

Reply:
I guess it comes down to what we want to do with the `_scriptUri` function below. Is there a valid default if this value is null?

Reply:
Probably no reasonable default, so if it is always going to be set then late is fine.